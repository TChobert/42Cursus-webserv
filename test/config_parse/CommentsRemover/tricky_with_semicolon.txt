value=12345#6
name=#john
# This comment starts with a hash
notacomment=###notacomment
key1=value1;this is a comment
key2=some value ; another comment
; full line comment with semicolon
key3=val;ue3
key4=val";"ue4
key5="val;ue5" ; comment here
