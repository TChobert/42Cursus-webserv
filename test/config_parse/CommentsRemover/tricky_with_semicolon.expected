value=12345
name=

notacomment=
key1=value1
key2=some value 

key3=val
key4=val";"ue4
key5="val;ue5" 
